Project-Aurora/Project-Aurora/Project-Aurora.csproj

```diff
@@ -122,6 +122,10 @@
     <Reference Include="IronPython.Wpf, Version=2.7.8.0, Culture=neutral, PublicKeyToken=7f709c5b713576e1, processorArchitecture=MSIL">
       <HintPath>..\packages\IronPython.2.7.8.1\lib\net45\IronPython.Wpf.dll</HintPath>
     </Reference>
+    <Reference Include="keyboard, Version=1.0.0.0, Culture=neutral, processorArchitecture=AMD64">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>.\keyboard.dll</HintPath>
```

Comment:
please rename this to something more descriptive.  I will have no idea what "keyboard.dll" does in a few days.

Reply:
I will change to our name.

Reply:
please remove the reference to the older dll